tests/macro_expansion/test_macro_expansion.py

```diff
@@ -96,7 +96,7 @@ def test_variadic(self):
 
         for def_string in [
             "eprintf(...)=fprintf(stderr, __VA_ARGS__)",
-                "eprintf(args...)=fprintf(stderr, args)"]:
+                "eprintf(argv...)=fprintf(stderr, argv)"]:
```

Comment:
I think this one should remain `args`: the parameter to `eprintf` in the C documentation is called `args`.